server/mainPr/src/main/java/com/team23/mainPr/Global/ExceptionHandler/ControllerExceptionHandler.java

```diff
@@ -11,22 +11,24 @@
 @RestControllerAdvice
 public class ControllerExceptionHandler {
 
-    @ExceptionHandler({MethodArgumentNotValidException.class})
-    @Order(value = 3)
+     @ExceptionHandler({MethodArgumentNotValidException.class})
```

Comment:
```suggestion
     @ExceptionHandler(MethodArgumentNotValidException.class)
```
이하의 클래스에도 위의 형태를 갖추는 것을 추천합니다.